<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AFCGUI</class>
 <widget class="RollupContents" name="AFCGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>100</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>320</width>
    <height>100</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>560</width>
    <height>100</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Liberation Sans</family>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>AFC</string>
  </property>
  <widget class="QWidget" name="settingsContainer" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>340</width>
     <height>111</height>
    </rect>
   </property>
   <property name="windowTitle">
    <string>Settings</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>3</number>
    </property>
    <property name="leftMargin">
     <number>2</number>
    </property>
    <property name="topMargin">
     <number>2</number>
    </property>
    <property name="rightMargin">
     <number>2</number>
    </property>
    <property name="bottomMargin">
     <number>2</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="mainLayout">
      <item>
       <widget class="ButtonSwitch" name="startStop">
        <property name="toolTip">
         <string>start/stop acquisition</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../../sdrgui/resources/res.qrc">
          <normaloff>:/play.png</normaloff>
          <normalon>:/stop.png</normalon>:/play.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="devicesRefresh">
        <property name="maximumSize">
         <size>
          <width>24</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Refresh indexes of available device sets</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../../sdrgui/resources/res.qrc">
          <normaloff>:/recycle.png</normaloff>:/recycle.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="trackerLabel">
        <property name="text">
         <string>Tracker</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="trackerDevice">
        <property name="minimumSize">
         <size>
          <width>55</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Tracker deviceset index</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="trackedLabel">
        <property name="text">
         <string>Tracked</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="trackedDevice">
        <property name="minimumSize">
         <size>
          <width>55</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Tracked deviceset index</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="devicesApply">
        <property name="maximumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Reinitialization of device data</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../../sdrgui/resources/res.qrc">
          <normaloff>:/arrow_left.png</normaloff>:/arrow_left.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="targetLayout">
      <item>
       <widget class="ButtonSwitch" name="hasTargetFrequency">
        <property name="toolTip">
         <string>Tracker target frequency auto adjust</string>
        </property>
        <property name="text">
         <string>T</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="ValueDial" name="targetFrequency" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>32</width>
          <height>16</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Liberation Mono</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="focusPolicy">
         <enum>Qt::StrongFocus</enum>
        </property>
        <property name="toolTip">
         <string>Tracker target tracker frequency in Hz</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="targetFrequencyUnits">
        <property name="text">
         <string>Hz</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="ButtonSwitch" name="transverterTarget">
        <property name="toolTip">
         <string>Use transverter frequency for tracker target frequency adjustment</string>
        </property>
        <property name="text">
         <string>X</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="toleranceLayout">
      <item>
       <widget class="QLabel" name="toleranceLabel">
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>&amp;plusmn;</string>
        </property>
        <property name="textFormat">
         <enum>Qt::RichText</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="ValueDial" name="toleranceFrequency" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>32</width>
          <height>16</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Liberation Mono</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="focusPolicy">
         <enum>Qt::StrongFocus</enum>
        </property>
        <property name="toolTip">
         <string>Tracker target frequency error tolerance (Hz)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="toleranceUnits">
        <property name="text">
         <string>Hz</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDial" name="targetPeriod">
        <property name="maximumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Tracker target frequency adjustment period (seconds)</string>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>120</number>
        </property>
        <property name="pageStep">
         <number>1</number>
        </property>
        <property name="value">
         <number>20</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="targetPeriodText">
        <property name="minimumSize">
         <size>
          <width>24</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Tracker target frequency adjustment period (seconds)</string>
        </property>
        <property name="text">
         <string>000</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="deviceTrack">
        <property name="maximumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Force tracker target frequency adjustment</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../../sdrgui/resources/res.qrc">
          <normaloff>:/arrow_left.png</normaloff>:/arrow_left.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="statusIndicator">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Idle</string>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel { background-color: gray; border-radius: 8px; }</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>RollupContents</class>
   <extends>QWidget</extends>
   <header>gui/rollupcontents.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ButtonSwitch</class>
   <extends>QToolButton</extends>
   <header>gui/buttonswitch.h</header>
  </customwidget>
  <customwidget>
   <class>ValueDial</class>
   <extends>QWidget</extends>
   <header>gui/valuedial.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../sdrgui/resources/res.qrc"/>
 </resources>
 <connections/>
</ui>
